lib/collection/src/operations/verification/update.rs

```diff
@@ -56,6 +56,18 @@ impl StrictModeVerification for DeleteVectors {
 }
 
 impl StrictModeVerification for SetPayload {
+    // TODO: Payload storage size limit
```

Comment:
this one can now be implemented in a different PR

I can take care of it later to get use to the `StrictModeVerification` infra :)

Reply:
Started in https://github.com/qdrant/qdrant/pull/5588